.changeset/brown-onions-carry.md

```diff
@@ -0,0 +1,5 @@
+---
+"@quiltt/core": patch
```

Comment:
can you regenerate this to bump everything? we keep versions in sync (could be good to add a CONTRIBUTING.md file to document this). For example: https://github.com/quiltt/quiltt-public/pull/133/files